File: C/D3/3 - cost/3.cpp
// Write a C program....

#include <stdio.h>

int main(){
    int a, cost;

    printf("Enter value of A: ");
    scanf("%d", &a);

     cost = a>20 ? (a>50 ? 100 : 40) : 10;
     printf("Cost is %d", cost);
}

File: Python/D13 - F/3/3.py
# Write a python program to reverse the given string.


str = input("Enter a string : ")

len = len(str)

str_direct=str[-1::-1]
print('direct method : ', str_direct)

print('-------------------------------------------')

# with for loop --------------------------------
str1=''

for i in range(len-1, -1, -1):
    str1+=str[i]

# str = str1
print('from for loop : ', str1)

print('-------------------------------------------')

# with list method --------------------------------

list1 = list(str)
list1.reverse()
rev_str =''.join(list1)

print('from list method : ', rev_str)